File: GEULGWI-BACKEND/src/main/java/com/lovedbug/geulgwi/core/domain/auth/dto/request/LoginRequest.java
package com.lovedbug.geulgwi.core.domain.auth.dto.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Builder
@NoArgsConstructor
@AllArgsConstructor
@Getter
public class LoginRequest {

    private String email;
    private String password;
}

File: GLIM-AOS/app/src/main/java/com/ssafy/glim/core/data/dto/request/LoginRequest.kt
package com.ssafy.glim.core.data.dto.request

import com.google.gson.annotations.SerializedName
import kotlinx.serialization.Serializable

@Serializable
data class LoginRequest(
    @SerializedName("email")
    val email: String,
    @SerializedName("password")
    val password: String
)